build/dockerfiles/Dockerfile

```diff
@@ -19,6 +19,13 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+FROM docker.io/node:12.20.1-alpine3.12 as dwtemplates
+RUN apk add --no-cache bash && npm install -g @eclipse-che/che-theia-devworkspace-handler
+COPY ./build/scripts /build/
+COPY ./devfiles /build/devfiles
+WORKDIR /build/
+RUN ./generate_devworkspace_templates.sh
+
```

Comment:
we're still missing instructions in https://github.com/eclipse-che/che-devfile-registry/blob/main/build/dockerfiles/rhel.Dockerfile

Reply:
Added in the latest fixup